Dockerfile

```diff
@@ -1,6 +1,8 @@
 FROM phusion/baseimage:0.9.16
 MAINTAINER Modulus <help@modulus.io>
 
+ENV REFRESHED_ON=12-15-2015
```

Comment:
Isn't this accomplished by just doing:

`docker build --no-cache`

Seems like an anti-pattern to add an environment variable just to force rebuild. I know you can use this while in dev to preserve upper layers, but for the published source?